tox.ini

```diff
@@ -8,6 +8,7 @@ setenv =
 deps =
     mock
     pytest
+    -rrequirements.txt
```

Comment:
What syntax is this? It kinda looks like a typo

Reply:
It runs pip to install the deps in requirements.txt